README.md

```diff
@@ -1591,6 +1592,12 @@ $ rm -rf .git/modules/submodulename
 
 ## Miscellaneous Objects
 
+### Copy a folder from one branch to another
+
+```sh
+$ (my-branch)$ git checkout <branch-you-want-the-directory-from> --<folder-name>
```

Comment:
You need a space between the dash and the carrot in  `--<folder-name>`. This also also works for a file from another branch; perhaps add a sentence clarifying that?

Reply:
sure